tools/build_env/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+Xvfb :10 -screen 0 1280x1024x24 &
```

Comment:
Oooooh really wish we could remove this...X is only needed for integration tests.